sphinx/source/rst_epilogue.txt

```diff
@@ -0,0 +1,14 @@
+.. |bokeh.models|       replace:: :ref:`bokeh.plotting <bokeh.plotting>`
+.. |bokeh.plotting|     replace:: :ref:`bokeh.plotting <bokeh.plotting>`
+.. |Color|              replace:: :py:class:`~bokeh.core.properties.Color`
+.. |DataSpec|           replace:: :py:class:`~bokeh.core.properties.DataSpec`
+.. |Document|           replace:: :py:class:`~bokeh.document.Document`
+.. |field|              replace:: :py:func:`~bokeh.core.properties.field`
+.. |figure|             replace:: py:func:`~bokeh.plotting.figure`
```

Comment:
```suggestion
.. |figure|             replace:: :py:func:`~bokeh.plotting.figure`
```

Reply:
Did copy and paste a bit too quickly here 😊